lib/beaker/hypervisor/docker.rb

```diff
@@ -4,6 +4,7 @@ class Docker < Beaker::Hypervisor
     def initialize(hosts, options)
       require 'docker'
       @options = options
+      @options[:set_env] = false
```

Comment:
In that case, the docker hypervisor should overwrite the whole 'configure' method to one that is more appropriate - instead of picking and choosing out of the default configure method.